root.js

```diff
@@ -5,6 +5,15 @@
  */
 'use strict';
 
+const fs = require('fs');
+
 module.exports = {
   LH_ROOT: __dirname,
+  /** @param {string} path */
+  readJson(path) {
+    // TODO: could be nice to accept paths relative to LH_ROOT ? this would still
+    // allow absolute paths.
+    // return JSON.parse(fs.readFileSync(path.resolve(__dirname, path), 'utf-8'));
```

Comment:
thoughts? this would simplify a lot of call sites.

Reply:
FWIW, I've found that when I have the choice of a `dirname` replacement and an equivalent to `LH_ROOT`, I end up usually reaching for the `LH_ROOT`, especially when the subdirectories are well separated and I have to go way up and back down again anyways :)

OTOH, I've also found as I've gotten used to them that dealing with `import.meta.url` and `fileURLToPath` to be increasingly not an impediment to writing or reading code. I wonder how much this appeals in a mass refactor as a way to centralize an annoying step, while it's no big deal authoring individual files. It sucks to lose json `require` (until import assertions [finally land](https://github.com/nodejs/node/pull/37375)), but in the past when I couldn't `require` a json file (needing to mutate it or whatever), it really has never been a big deal to throw in a `JSON.parse(fs.readFileSync)` line.

Not objecting to adding any helpful utilities, just saying we should maintain an open mind as the codebase continues to adapt to full ESM :)

Reply:
> especially when the subdirectories are well separated and I have to go way up and back down again anyways

Yes, definitely the situation where LH_ROOT is preferred. Many of the usages here though are for a local file in the same directory or an immediate neighbor, in which case the respecifying of the entire path to the subdir is suboptimal IMO.

Reply:
how about `readJson(path, baseDirOrImportMeta=LH_ROOT)` ?

:D